internal/olm/operator/internal/registry_pod_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package olm
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+// newFakeClient() returns a clientset
+func newFakeClient() kubernetes.Interface {
+	return fake.NewSimpleClientset()
+}
+
+func TestCreatePod(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Test registry pod")
+}
+
+var _ = Describe("Creation", func() {
+	rp := &RegistryPod{
```

Comment:
This assignment should probably be in a `BeforeEach` so that one test doesn't affect another.
```go
var rp *RegistryPod

BeforeEach(func() {
    rp = &RegistryPod{ ... }
})
```